.github/workflows/report_quiz/report_quiz.sh

```diff
@@ -41,12 +41,12 @@ send_poll(){
 # Propose a quiz
 propose_quiz(){
     ret=$(get_quiz_questions_answers)
-    echo $ret | jq -r '[.question, .A, .B, .C, .D] | @tsv' | \
+    echo $ret | jq -r '[.question, .A, .B, .C, .D, .E, .F] | @tsv' | \
     while IFS=$'\t' read -r question A B C D; do
```

Comment:
If you add options E and F at line 44, don't forget to add it also on this line !
```suggestion
    while IFS=$'\t' read -r question A B C D E F; do
```
otherwise, those options will not be available !